python/paddle/fluid/layers/tensor.py

```diff
@@ -436,34 +445,36 @@ def fill_constant_batch_size_like(input,
                                   input_dim_idx=0,
                                   output_dim_idx=0):
     """
-    ${comment}
-
-    It also sets *stop_gradient* to True.
+    This OP creates a Tesnor accroding the shape and dtype, and initializes the
+    Tensor with the constants provided in ``value``. When the input is LoDTensor
+    and the input_dim_idx is 0, the output_dim_idx dimension is set to the value
+    of the batch_size input by the input, the Stop_gradient attribute of the created
+    Tensor is False by default.
 
     Args:
-        input(${input_type}): ${input_comment}.
-
-        shape(${shape_type}): ${shape_comment}.
-
-        dtype(${dtype_type}): ${dtype_comment}.
-
-        value(${value_type}): ${value_comment}.
-
-        input_dim_idx(${input_dim_idx_type}): ${input_dim_idx_comment}.
-
-        output_dim_idx(${output_dim_idx_type}): ${output_dim_idx_comment}.
+        input(Variable): Tensor which data type is float32, float64, int32 and int64.
+        shape(list): The shape of Tensor to be created, Tensor's shape may be changed
+            according the input.
+        dtype(np.dtype|core.VarDesc.VarType|str): The data type of created Tensor which
+            can be float32, float64, int32, int64.
+        value(float|int): The constant value used to initialize the Tensor to be created. 
+        input_dim_idx(int): When the value is 0 and the input is LoDTensor, the output_dim_idx
+            dimension of the created Tensor is set to the batch_size value of input.
+            The default value is 0.
+        output_dim_idx(int): Used to specify which dimension of Tensor is created to be set
+            the value of batch_size of input Tensor. The default value is 0.
 
     Returns:
-        ${out_comment}.
+        Variable: the created Tensor which data type si dtype.
```

Comment:
the created tensor whose data type is dtype

Reply:
Done